docs/getting-started/10-create-function.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Create a Function
+type: Getting Started
+---
+
+Let's now repeat the microservice flow for a Function. This guide shows how you can create a simple Function (`orders-function`) with the same logic as the one in the microservice. In further guides, you will expose the Function, bind it to the Redis storage, and subscribe it to the `order.deliverysent.v1` event type from Commerce mock.
```

Comment:
```suggestion
Let's now repeat the microservice flow for the Function. This guide shows how you can create a simple Function (`orders-function`) with the same logic as the one in the microservice. In further guides, you will expose the Function, bind it to the Redis storage, and subscribe it to the `order.deliverysent.v1` event type from Commerce mock.
```
I believe here we mean `Function` as the concept, not an instance, just like `the microservice`.